hack/ci/create_rbac_role

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Create ClusterRole
+cat <<EOF | kubectl create -f -
```

Comment:
Well it's more convenient as a bash script for immediate purposes so we can test this out with jenkins. Perhaps we can move this into test/e2e/utils.go later.